qt/widgets/common/src/MantidHelpWindow.cpp

```diff
@@ -112,9 +112,6 @@ MantidHelpWindow::MantidHelpWindow(QWidget *parent, const Qt::WindowFlags &flags
   }
 }
 
-/// Destructor does nothing.
-MantidHelpWindow::~MantidHelpWindow() { this->shutdown(); }
```

Comment:
Could you help me understand why we no longer need to call `shutdown`?

Reply:
The `shutdown` method will close the `g_helpWindow`, and this deletes it as the `WA_DeleteOnClose` attribute is set. I've made a change so that `MantidHelpWindow` is now the parent of `g_helpWindow`, so whenever `MantidHelpWindow` is destructed, the `g_helpWindow` will be closed and destructed without needing to do it explicitly.

Reply:
Ah yes okay. And now that `g_helpWindow` is a `QPointer` we can now detected if it has been deleted 👍🏻 Good stuff.